addons/website/views/website_views.xml

```diff
@@ -220,6 +256,11 @@
                             type="object" attrs="{'invisible': [('page_ids', '=', [])]}"/>
                     </header>
                 </sheet>
+                <page name="inherit_children" position="after">
+                    <page string="Website Pages" attrs="{'invisible': [('page_ids', '=', [])]}">
+                        <field name="page_ids"/>
```

Comment:
@JKE-be 
Since it is a O2M relation, it shows in a list all the related pages. Clicking on it will pop up the form view of the page in readonly. Is there a way to create a link to access the website.page's form view directly?

Reply:
Like records in others tabs, clicking on records should open an editable form view, not readonly